packages/pds/src/feed-gen/with-friends.ts

```diff
@@ -0,0 +1,84 @@
+import AppContext from '../context'
+import { QueryParams as SkeletonParams } from '../lexicon/types/app/bsky/feed/getFeedSkeleton'
+import { paginate } from '../db/pagination'
+import { countAll } from '../db/util'
+import { AlgoHandler, AlgoResponse } from './types'
+import { FeedKeyset } from '../app-view/api/app/bsky/util/feed'
+
+const handler: AlgoHandler = async (
```

Comment:
There's a TODO on this one to sort out an issue it has when running with sqlite.